co-demos/apiviz-frontend
# integration backoffice  (#8)

* light changes on map

* towards a JSON editor for back office

* towards a JSON editor for back office

* fixing backoffice navigation + tabs

* fixing backoffice navigation + tabs

* testing several json editors

* fixing minor bug at /backoffice root + starting with a json editor

* first instanciation of cookie check for auth middleware

* minor changes to navbar and footer (dynamic variables)

* update documentation

* adding some buttons and modals in backoffice

* adding warnings in backoffice doc edition

* simplifying dispatch - no exception for navbar links

* minor fixes on cards UXUI in backoffice

* added DELETE function and fixinng some stuff in POST / update doc function

* update documentation

* adding translations for filters

* fixing stuff on footer + cleaning

* cleaning css filters' choices list

* cleaning css filters' choices list

* new apiviz instance form + plugins to backend
